README.md

```diff
@@ -6,22 +6,18 @@
 [![Quix on Twitter](https://img.shields.io/twitter/url?label=Twitter&style=social&url=https%3A%2F%2Ftwitter.com%2Fquix_io)](https://twitter.com/quix_io)
```

Comment:
Could you also please add a badge for YouTube. Badges should be in the order:

1. Docs
2. Community Slack
3. YouTube
4. LinkedIn
5. X

For badge colours, I'd make `Docs` green and `YouTube` bright red